cms/static/js/views/content_group_details.js

```diff
@@ -21,10 +31,65 @@ define([
 
         initialize: function() {
             this.template = this.loadTemplate('content-group-details');
+            this.listenTo(this.model, 'change', this.render);
         },
 
         render: function() {
-            this.$el.html(this.template(this.model.toJSON()));
+           var attrs = $.extend({}, this.model.attributes, {
+                usageCountMessage: this.getUsageCountTitle(),
+                outlineAnchorMessage: this.getOutlineAnchorMessage(),
+                index: this.model.collection.indexOf(this.model)
+            });
+            this.$el.html(this.template(attrs));
+            return this;
+        },
+
+        showContentGroupUsages: function(event) {
+            if (event && event.preventDefault) { event.preventDefault(); }
+            this.model.set('showContentGroupUsages', true);
+        },
+
+        hideContentGroupUsages: function(event) {
+            if (event && event.preventDefault) { event.preventDefault(); }
+            this.model.set('showContentGroupUsages', false);
+        },
+
+        getUsageCountTitle: function () {
+            var count = this.model.get('usage').length, message;
+
+            if (count === 0) {
+                message = gettext('Not in Use');
+            } else {
+                message = ngettext(
+                    /*
+                        Translators: 'count' is number of units that the group
+                        configuration is used in.
+                    */
+                    'Used in %(count)s unit', 'Used in %(count)s units',
+                    count
+                );
+            }
+
+            return interpolate(message, { count: count }, true);
+        },
+
+        getOutlineAnchorMessage: function () {
+            var message = gettext(
+                    /*
+                        Translators: 'outlineAnchor' is an anchor pointing to
+                        the course outline page.
+                    */
+                    'This Content Group is not in use. Start by adding a content group to any Unit via the %(outlineAnchor)s.'
```

Comment:
My suggested rewrite for the UI text would be: "This content group is not in use. Add a content group to any unit from the Course Outline".